pkg/diff/diff.go

```diff
@@ -690,7 +686,7 @@ func generateDML(tp string, data map[string]*dbutil.ColumnData, keys []*model.Co
 		sql = fmt.Sprintf("REPLACE INTO `%s`.`%s`(%s) VALUES (%s);", schema, table.Name, strings.Join(colNames, ","), strings.Join(values, ","))
 	case "delete":
 		kvs := make([]string, 0, len(keys))
-		for _, col := range keys {
+		for _, col := range table.Columns {
```

Comment:
for example:
have two chunk: chunk1's range is a = 1 and b <= 1, chunk2's range is a = 1 and b > 1, and a is the primary key, b is column with index.
in source have one row a=1, b = 0, and in target have a wrong row  a = 1, b =2
when compare chunk1 ,will generate a fix sql sql1: replace into test.t1 values(1, 0),
when compare chunk2, will generate a fix sql sql2 if where only contain pk/uk: delete test.t1 where a = 1

when user execute these fix sqls, and sql1 is before sql2 will delete this row.
so need contain all the column's value in delete sql.

Reply:
@WangXiangUSTC so, can we remove the `keys` arg now?

Reply:
removed